ui/components/Footer/Footer.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import { useStyles } from "./Footer.styles";
+import FavoriteIcon from "@mui/icons-material/Favorite";
+const FooterComponent = () => {
+  const classes = useStyles();
+
+  const handleComClick = () => {
+    if (typeof window !== "undefined") {
+      const L = window.open("https://layer5.io", "_blank");
+      L.focus();
+    }
+  };
+  return (
+    <footer className={classes.footer}>
+      <Typography variant="body2" align="center" color="textSecondary">
+        <span onClick={handleComClick} className={classes.footerText}>
+          Built with
+          <FavoriteIcon className={classes.footericon} /> by the Layer5 Community

```

Comment:
Remember we discussed that each icon should have a fixed height and width in order for it to scale properly in safari, we have to create a custom icon wrapper component which will do this for us  and use it here.

Reply:
No need to do it in this pr, but just giving a heads up

Reply:
Oh yes, i remember will do that eventually.

Reply:
Thank you